analyses/molecular-subtyping-ATRT/Archive/run-molecular-subtyping-ATRT.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Chante Bethell for CCDL 2019
```

Comment:
```suggestion
# Chante Bethell for CCDL 2019, Zhuanzhuang Geng 2023
```

Reply:
This is the bash script in archive folder. Should I make change here?

Reply:
oh whoops - no you should delete this one fully. Sorry, didn't realize you had two in here!